torch/csrc/distributed/c10d/control_collectives/StoreCollectives.hpp

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include <c10/macros/Macros.h>
+#include <torch/csrc/distributed/c10d/Store.hpp>
+#include <torch/csrc/distributed/c10d/control_collectives/ControlCollectives.hpp>
+
+namespace c10d {
+
+class TORCH_API StoreCollectives : public ControlCollectives {
```

Comment:
StoreControlCollectives?

Reply:
I thought about that but figured Store would be clear enough -- it wouldn't make sense to use the Store API for dataplane operations